theano/tensor/nnet/nnet.py

```diff
@@ -384,6 +384,7 @@ def c_code(self, node, name, inp, out, sub):
             }
         }
 
+        #pragma omp parallel for schedule(static, 32)
```

Comment:
Thanks for the PR, but I think this isn't enought. You should make the ops inherit from theano.gof.OpenMPOp instead of theano.gof.Op. This will allow Theano to pass the right compiler parameter to enable it and will respect the flag openmp={True,False}.

Did you do some timing? I think with the current state, it should always be disabled as we don't tell g++ to enable openmp.